.scala-steward.conf

```diff
@@ -4,3 +4,8 @@ postUpdateHooks = [{
   groupId = "io.opentelemetry.semconv",
   artifactId = "opentelemetry-semconv"
 }]
+
+updates.allowPreReleases = [
+  # semconv is always published with `-alpha` suffix
+  {groupId = "io.opentelemetry.semconv", artifactId = "opentelemetry-semconv"}
+]
```

Comment:
We previously discussed this in https://github.com/typelevel/otel4s/pull/336#issuecomment-1758375899. I don't think it will fix updates (or if so, then its a steward bug I think). But I suspect that something else is wrong.

Reply:
Can it be caused by non-trivial scope of the dependency?

```scala
libraryDependencies += 
  "io.opentelemetry.semconv" % "opentelemetry-semconv" % OpenTelemetrySemConvVersion % "compile-internal" intransitive (),
```

Reply:
Ah yeah, maybe. Maybe the `intransitive()` is causing the dependency not to be exported to steward?

Reply:
It might be a steward bug; I have to update the `-alpha` dependencies manually in some of our other projects (e.g. https://github.com/Dwolla/dwolla-otel-natchez/commit/74d6a89483a4a614c2a9065f6486a5926806034c)